courses-app/src/components/Courses/Courses.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState } from 'react';
+
+import CoursesCard from './components/CourseCard/CourseCard';
+import SearchBar from './components/SearchBar/SearchBar';
+import Button from '../../common/Button/Button';
+import {
+	ADD_NEW_BTN_TEXT,
+	BTN_TYPE_BUTTON,
+	mockedCoursesList,
+} from '../../constants';
+import getCourseAuthors from '../../helpers/getCourseAuthors';
+
+import './Courses.css';
+
+function Courses(props) {
+	const [renderCoursesList, setRenderCourseList] = useState(mockedCoursesList);
```

Comment:
Naming.

renderCoursesList it's a name for function (name has verb).
Here we have a variable so it should be _coursesList_